client/src/main/webjar/organization-form/organization-form.html

```diff
@@ -35,9 +39,11 @@
             </label>
             <input name="organizationEmail"
                    ng-model="organizationForm.organization.contactEmail"
+                   md-maxlength="organizationForm.config.emailMaxLength"
                    type="email">
             <div ng-messages="organizationForm.orgForm.organizationEmail.$error">
                 <div ng-message="email">Organization email must be a valid email</div>
+                <div ng-message="md-maxlength">Organization email is too long</div>
```

Comment:
This isn't visible (at least not in chrome), because type=email is enforcing a smaller number of characters than what I thought I had read on wikipedia.